sites/industrial-lasers/server/templates/index.marko

```diff
@@ -53,15 +53,15 @@ $ const adSlots = {
       <div class="row">
         <div class="col-lg-6 mb-block">
           <endeavor-content-query-section-list
-            limit=4
+            limit=3
             section-alias="micromachining"
             header={ title: "Micromachining", href: "/micromachining" }
             native-x={ placement: 'list1', aliases: ['micromachining'], index: 3 }
```

Comment:
The `native-x.index` needs to change from `3` to `2`.

I know it's confusing, but NX's positioning starts at `0`. So if you have a content limit of 3 (and you want NX to be the last item), then the NX index would need to be 2. 

So, when limit=3, index=2
limit=4, index=3
... and so on ...

Reply:
Not confusing, I just didn't know. Do we have documentation over these items anywhere?

Reply:
Unfortunately we haven't documented the individual component parameters/input yet... @solocommand - put this on a list somewhere?

Reply:
At this point, I'm making a lot of guesses and learning along the way :D